src/BasisTheory.net/Proxies/Requests/ProxyCreateRequest.cs

```diff
@@ -0,0 +1,50 @@
+using System;
+using System.Collections.Generic;
+using System.Text.Json.Serialization;
+using BasisTheory.net.Applications.Entities;
+using BasisTheory.net.Proxies.Entities;
+using Newtonsoft.Json;
+
+namespace BasisTheory.net.Proxies.Requests
+{
+    public class ProxyCreateRequest
+    {
+        [JsonProperty("name")]
+        [JsonPropertyName("name")]
+        public string Name { get; set; }
+
+        [JsonProperty("destination_url")]
+        [JsonPropertyName("destination_url")]
+        public string DestinationUrl { get; set; }
+
+        [JsonProperty("request_reactor_id")]
+        [JsonPropertyName("request_reactor_id")]
+        [Obsolete("RequestReactorId has been deprecated in favor of RequestTransform")]
+        public Guid? RequestReactorId { get; set; }
+
+        [JsonProperty("response_reactor_id")]
+        [JsonPropertyName("response_reactor_id")]
+        [Obsolete("ResponseReactorId has been deprecated in favor of ResponseTransform")]
+        public Guid? ResponseReactorId { get; set; }
+
+        [JsonProperty("request_transform")]
+        [JsonPropertyName("request_transform")]
+        public ProxyTransform RequestTransform { get; set; }
+
+        [JsonProperty("response_transform")]
+        [JsonPropertyName("response_transform")]
+        public ProxyTransform ResponseTransform { get; set; }
+
+        [JsonProperty("application")]
+        [JsonPropertyName("application")]
+        public Application Application { get; set; }
```

Comment:
You pass in a full Application when creating a proxy? Not just the id?

Reply:
Yea, but I only parse the application ID when create/update. I am hoping in the future we allow creating an application behind the scenes.